### ISSUE TITLE
astmonkey 0.3 is incompatible with mutpy

### ISSUE THREAD

**Issue by dunossauro** [mutpy/mutpy#33]
New version of [astmonkey](https://github.com/mutpy/astmonkey/commit/49054146e349f553c2d255956ef4317161bb9aa9)  is not compatible with mutpy.

```
Traceback (most recent call last):
  File "/builds/../../.tox/check_tests/bin/mut.py", line 7, in <module>
    commandline.main(sys.argv)
  File "/builds/../../.tox/check_tests/lib/python3.6/site-packages/mutpy/commandline.py", line 9, in main
    run_mutpy(parser)
  File "/builds/../../.tox/check_tests/lib/python3.6/site-packages/mutpy/commandline.py", line 59, in run_mutpy
    mutation_controller.run()
  File "/builds/../../.tox/check_tests/lib/python3.6/site-packages/mutpy/controller.py", line 66, in run
    self.run_mutation_process()
  File "/builds/../../.tox/check_tests/lib/python3.6/site-packages/mutpy/controller.py", line 85, in run_mutation_process
    self.mutate_module(target_module, to_mutate, total_duration)
  File "/builds/../../.tox/check_tests/lib/python3.6/site-packages/mutpy/utils.py", line 259, in __call__
    result = self.method(*args, **kwargs)
  File "/builds/../../.tox/check_tests/lib/python3.6/site-packages/mutpy/controller.py", line 134, in mutate_module
    self.run_tests_with_mutant(total_duration, mutant_module, mutations, coverage_result)
  File "/builds/../../.tox/check_tests/lib/python3.6/site-packages/mutpy/utils.py", line 259, in __call__
    result = self.method(*args, **kwargs)
  File "/builds/../../.tox/check_tests/lib/python3.6/site-packages/mutpy/controller.py", line 200, in run_tests_with_mutant
    self.update_score_and_notify_views(result, timer.duration)
  File "/builds/../../.tox/check_tests/lib/python3.6/site-packages/mutpy/controller.py", line 220, in update_score_and_notify_views
    self.update_killed_mutant(result, mutant_duration)
  File "/builds/../../.tox/check_tests/lib/python3.6/site-packages/mutpy/controller.py", line 235, in update_killed_mutant
    self.notify_killed(duration, result.killer, result.exception_traceback, result.tests_run)
  File "/builds/../../.tox/check_tests/lib/python3.6/site-packages/mutpy/views.py", line 30, in <lambda>
    return lambda *args, **kwargs: self.notify_all_views(notify, *args, **kwargs)
  File "/builds/../../.tox/check_tests/lib/python3.6/site-packages/mutpy/views.py", line 25, in notify_all_views
    attr(*args, **kwargs)
  File "/builds/../../.tox/check_tests/lib/python3.6/site-packages/mutpy/views.py", line 198, in killed
    exception_traceback=exception_traceback,
  File "/builds/../../.tox/check_tests/lib/python3.6/site-packages/mutpy/views.py", line 260, in end_mutation
    'mutant_code': codegen.to_source(self.current_mutation['mutant']),
  File "/builds/../../.tox/check_tests/lib/python3.6/site-packages/mutpy/codegen.py", line 8, in to_source
    return astmonkey_to_source(node=node, indent_with=indent_with)
  File "/builds/../../.tox/check_tests/lib/python3.6/site-packages/astmonkey/visitors.py", line 134, in to_source
    parent_child_transformer.visit(node)
  File "/builds/../../.tox/check_tests/lib/python3.6/site-packages/astmonkey/transformers.py", line 9, in visit
    self._process_field(node, field, value)
  File "/builds/../../.tox/check_tests/lib/python3.6/site-packages/astmonkey/transformers.py", line 24, in _process_field
    self._process_child(item, node, field, index)
  File "/builds/../../.tox/check_tests/lib/python3.6/site-packages/astmonkey/transformers.py", line 29, in _process_child
    self.visit(child)
  File "/builds/../../.tox/check_tests/lib/python3.6/site-packages/astmonkey/transformers.py", line 9, in visit
    self._process_field(node, field, value)
  File "/builds/../../.tox/check_tests/lib/python3.6/site-packages/astmonkey/transformers.py", line 24, in _process_field
    self._process_child(item, node, field, index)
  File "/builds/../../.tox/check_tests/lib/python3.6/site-packages/astmonkey/transformers.py", line 31, in _process_child
    child.parents.append(parent)
AttributeError: 'alias' object has no attribute 'parents'
```

**Comment by reebr** [comment]
Having the same problem here.

**Comment by williangl** [comment]
I'm having the same problem too

**Comment by phihos** [comment]
Thanks for pointing that out. I just released a hotfix for astmonkey (v0.3.1). Please update and tell me whether this bug still persists.

**Comment by phihos** [comment]
I am closing this issue now. If it persists feel free to comment here and will reopen it.
